src/IoC/PersistenceExtensions.cs

```diff
@@ -232,7 +232,13 @@ public static IServiceCollection AddPersistence(this IServiceCollection services
                 .AddTransient<IRepository<BomDetail, int>, BomDetailRepository>()
                 .AddTransient<IRepository<PcasChange, int>, EntityFrameworkRepository<PcasChange, int>>(
                     r => new EntityFrameworkRepository<PcasChange, int>(r.GetService<ServiceDbContext>()
-                        ?.PcasChanges));
+                        ?.PcasChanges))
+                .AddTransient<IRepository<BomDetail, int>, EntityFrameworkRepository<BomDetail, int>>(
```

Comment:
is this bom detail repo meant to be added?

Reply:
This was a merge error where it was moved. Back in its regular space now in latest commit.